Expose a way to exclude dependencies (#664)
ansible/ansible-builder: docs/collection_metadata.rst

::

  ansible-builder introspect ~/

.. _python_deps:

Python Dependencies
^^^^^^^^^^^^^^^^^^^

Ansible Builder combines all the Python requirements files from all collections into a single file.

Certain package names are specifically *ignored* by ``ansible-builder``, meaning that Ansible Builder
does not include them in the combined file of Python dependencies, even if a collection lists them as

> dependencies. These include test packages and packages that provide Ansible itself. The full list can

Reviewer: ```suggestion
dependencies. These include Ansible required packages and also test packages. The full list can
```

Author: "Ansible required packages" doesn't apply here, IMO.
